src/webui/www/private/views/log.html

```diff
@@ -272,7 +272,7 @@
         };
 
         const getSyncLogDataInterval = () => {
-            return customSyncLogDataInterval ? customSyncLogDataInterval : serverSyncMainDataInterval;
+            return serverSyncMainDataInterval;
```

Comment:
CodeQL report `customSyncLogDataInterval` always evaluates to `false` which I inspected and verified it.

Reply:
So what was `customSyncLogDataInterval` supposed to be for? Isn't it really used anywhere else in the code?

Reply:
I don't know, I wasn't the author.

Reply:
The code looked as if `customSyncLogDataInterval` was expected to be set somewhere else in the code, so here it was checked whether it was set, and otherwise the fallback option was used. So there was my second question, is `customSyncLogDataInterval` really not mentioned anywhere else in the code?

Reply:
It is used at line 185 and 247 but both set it to `null`.

Reply:
> It appeared at line 185 and 247 but both set it to `null`.

Is it a bad idea to delete them too so that they don't confuse anyone?
By the way, shouldn't the analyzer have complained about such "declared but never used variables"?

Reply:
>Is it a bad idea to delete them too so that they don't confuse anyone?

No but I don't want to refactor it. I only do the minimum to 'fix' the analyzer warnings. The original author might have plans for using it.
I would welcome anyone to clean it up *outside* of this PR.

Reply:
> > Is it a bad idea to delete them too so that they don't confuse anyone?
> 
> No but I don't want to touch it as I only do the minimum to 'fix' the analyzer warnings

IMO, in this case, it is better not to touch it at all than to "fix" the analyzer warnings, and not the problems that are behind these warnings. Having this warning visible, we will get a better chance that someone will notice it sometime and fix the underlying problem.

Reply:
OK, done.